openslides_backend/shared/mixins/user_create_update_permissions_mixin.py

```diff
@@ -307,7 +348,7 @@ def check_group_B(
             or locked_from_inside
         ) and fields:
             meeting_ids = self._meetings_from_group_B_fields_from_instance(instance)
-            if diff := meeting_ids - self.permstore.user_meetings:
+            if (diff := meeting_ids - self.permstore.user_meetings) and diff != {None}:
```

Comment:
Why is it possible for None to be in the set? Seems broken to me

Reply:
Done